nilrt_snac/_configs/_firewall_config.py

```diff
@@ -0,0 +1,141 @@
+import argparse
+import subprocess
+
+from nilrt_snac._configs._base_config import _BaseConfig
+
+from nilrt_snac import logger
+from nilrt_snac.opkg import opkg_helper
+
+def _check_target(policy: str, expected: str = "REJECT") -> bool:
+    "Verifies firewall-cmd --policy=POLICY --get-target matches what is expected."
+
+    actual: str = subprocess.getoutput(
+        f"firewall-cmd --permanent --policy={policy} --get-target")
+    if expected == actual:
+        return True
+    logger.error(f"ERROR: policy {desc} target: expected {expected}, observed {actual}")
```

Comment:
Is `desc` defined? What's the output when you run `verify` before `configure`?

Reply:
Hm, it bails because firewall-cmd isn't installed, but yes, good catch. Will fix.